python/lsst/ts/mtaos/model.py

```diff
@@ -423,6 +427,45 @@ def get_dof_lv(self) -> np.ndarray:
         """
         return self.ofc.lv_dof
 
+    def get_visit_ids(self) -> tuple[int, int | None]:
```

Comment:
I can see the benefits of having `extra_id` return value be `None` however, this will create an issue when publishing the value as integer values cannot be `None`.